tempest2/src/test/kotlin/app/cash/tempest2/AsyncLogicalDbBatchTest.kt

```diff
@@ -107,6 +133,21 @@ class AsyncLogicalDbBatchTest {
     assertThat(items).containsAll(albumTracks)
   }
 
+  @Disabled
```

Comment:
also want to add a `// Disabled pending https://github.com/cashapp/tempest/issues/124` here?